Tests/Fixtures/UnitTestFixtures.swift

```diff
@@ -0,0 +1,162 @@
+//
+//  Copyright (c) 2020-2023 MobileCoin. All rights reserved.
+//
+@testable import MobileCoin
+import XCTest
+
+enum UnitTestFixtures {
+    static let invalidConsensusUrl = "mc://invalid.mobilecoin.com"
+    static let invalidFogUrl = "fog://invalid.mobilecoin.com"
+
+    static func failingHttpNetworkConfig(
+        _ fogUrlLoadBalancer: SequentialUrlLoadBalancer<FogUrl>,
+        _ consensusUrlLoadBalancer: SequentialUrlLoadBalancer<ConsensusUrl>
+    ) throws -> NetworkConfig {
+        let attestation = Attestation(
+            mrEnclaves: [],
+            mrSigners: [])
+
+        let attestationConfig = NetworkConfig.AttestationConfig(
+            consensus: attestation,
+            fogView: attestation,
+            fogKeyImage: attestation,
+            fogMerkleProof: attestation,
+            fogReport: attestation)
+
+        var networkConfig = try NetworkConfig.make(
+            consensusUrlLoadBalancer: consensusUrlLoadBalancer,
+            fogUrlLoadBalancer: fogUrlLoadBalancer,
+            attestation: attestationConfig,
+            transportProtocol: .http)
+            .get()
+
+        networkConfig.httpRequester = MockHttpRequester()
+        return networkConfig
+    }
+
+    static func createFogViewConnection(
+        using fogUrlLoadBalancer: SequentialUrlLoadBalancer<FogUrl>
+    ) throws -> FogViewConnection {
+        let networkConfig = try failingHttpNetworkConfig(
+            fogUrlLoadBalancer,
+            UrlLoadBalancerFixtures().invalidConsensusUrlBalancer)
+
+        fogUrlLoadBalancer.rotationEnabled = false
```

Comment:
why do you set this to false here and then true later ?

Reply:
The initial creation of the service was rotating as a side effect during the creation - so to make sure that we're detecting rotation as a result of the attempted call, the rotation is disabled during service creation, and enabled for the test call (this only applies to testing)